lib/web_ui/lib/src/engine/platform_dispatcher/view_focus_binding.dart

```diff
@@ -16,7 +16,7 @@ final class ViewFocusBinding {
   ///
   /// DO NOT rely on this bit as it will go away soon. You're warned :)!
   @visibleForTesting
-  static bool isEnabled = false;
+  static bool isEnabled = true;
```

Comment:
@ditman we might want to reland these changes but with this one set to false. Not sure what's breaking on the framework side but don't think the rest of these changes is related.

Reply:
Will take a look!

Reply:
Reland posted: https://github.com/flutter/engine/pull/53537 /cc @tugorez